cardigan/stories/components/Promos.js

```diff
@@ -95,36 +91,66 @@ const EventPromoExample = () => {
+    'squareImage': {'contentUrl': 'https://wellcomecollection.cdn.prismic.io/wellcomecollection/318dba668b46078bd957578fa5fc3b2f9b86c5a0_sdp_20181009_0007.jpg', 'width': 3200, 'height': 3200, 'alt': 'Photograph showing a woman giving a talk in the Viewing Room at Wellcome Collection. She is stood at the front of the room looking at a wall mounted television screen. In the foreground are the backs of the heads of the audience.', 'tasl': {'title': 'Exploring Research event', 'author': 'Steven Pocock', 'sourceName': 'Wellcome Collection', 'sourceLink': null, 'license': 'CC-BY-NC', 'copyrightHolder': null, 'copyrightLink': null}, 'crops': {}},
+    'widescreenImage': {'contentUrl': 'https://wellcomecollection.cdn.prismic.io/wellcomecollection/1689f6e5ead8d3a228d802256213e0998b15b7a2_sdp_20181009_0007.jpg', 'width': 3200, 'height': 1800, 'alt': 'Photograph showing a woman giving a talk in the Viewing Room at Wellcome Collection. She is stood at the front of the room looking at a wall mounted television screen. In the foreground are the backs of the heads of the audience.', 'tasl': {'title': 'Exploring Research event', 'author': 'Steven Pocock', 'sourceName': 'Wellcome Collection', 'sourceLink': null, 'license': 'CC-BY-NC', 'copyrightHolder': null, 'copyrightLink': null}, 'crops': {}},
+    'labels': [{'url': null, 'text': 'Seminar'}].concat(activeAudiences).concat(activeInterpretations),
+    'place': {'id': 'WoLtUioAACkANrUM', 'title': 'Viewing Room', 'contributors': [], 'body': [], 'labels': [], 'level': 2, 'capacity': 20, 'information': [{'type': 'paragraph', 'text': 'We’ll be in the Viewing Room. It’s next to the Library entrance on level 2, which you can reach by taking the lift or the stairs.', 'spans': []}]},
+    'audiences': [],
+    'bookingInformation': null,
+    'bookingType': 'First come, first served',
+    'cost': null,
+    'format': {'id': 'WlYVBiQAACcAWcu9', 'title': 'Seminar', 'description': null},
+    'policies': [{'id': 'W3RLAikAACcAF2oO', 'title': 'Limited spaces available', 'description': [{'type': 'paragraph', 'text': 'You can claim your place for this event 15 minutes before it starts. Spaces are first come, first served and may run out if we are busy.', 'spans': []}]}],
+    'isDropIn': false,
+    'series': series ? eventSeries : [],
+    'schedule': schedule ? eventSchedule : [],
+    'eventbriteId': '',
+    'isCompletelySoldOut': false,
+    'ticketSalesStart': null,
+    'times': [{'range': {'startDateTime': new Date('2018-10-23T17:00:00.000Z'), 'endDateTime': new Date('2018-10-23T18:30:00.000Z')}, 'isFullyBooked': isFullyBooked}],
+    'displayStart': new Date('2018-10-23T17:00:00.000Z'),
+    'displayEnd': new Date('2018-10-23T18:30:00.000Z'),
+    'dateRange': {'firstDate': new Date('2018-10-23T17:00:00.000Z'), 'lastDate': new Date('2018-10-23T18:30:00.000Z'), 'repeats': 1},
+    'isPast': false,
+    'isRelaxedPerformance': false};
```

Comment:
I wonder how we might get these mocks in easier?
Maybe running a fetch => parse => save as the models wont be changing that often now?